### ISSUE TITLE
Invalid CamelCaseVariableName violation on prefixed properties

### ISSUE THREAD

**Issue by ravage84** [phpmd/phpmd#261]
It seems that the CamelCaseVariableName rule reports static acess to underscore prefixed properties as violation in PHPMD 2.2.0.

> The variable **$_colorArray** is not named in camelCase.

`````php
if (!in_array($colorName, self::$_colorArray)) {
````

This should not be reported as violation against the CamelCaseVariableName rule but rather against the CamelCasePropertyName rule.

**Comment by manuelpichler** [comment]
Does this issue still exist? Just tested it with the following code:

```php
<?php
class MyClass{
    function myFunction() {
        if (!in_array($colorName, self::$_colorArray)) { }
    }
}
```

and got no error

**Comment by ravage84** [comment]
@manuelpichler have a look at  #277

**Comment by steverobbins** [comment]
Having the same issue.  Is this going to get merged in?

**Comment by manuelpichler** [comment]
Fixed
